Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -8,6 +8,7 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
   // MARK: - Properties
 
   private let viewModel = PledgeAmountCellViewModel()
+  private let dataSource = PledgeDataSource()
```

Comment:
How about we put this `dataSource` on the `PledgeTableViewController` and maybe do a bit of cleanup so that there would be 2 data sources:

* `pledgeDataSource` - responsible for summary, amount, shipping, etc.
* `paymentMethodsDataSource` - responsible for payment methods

Then when the cell gets displayed in `tableView(_: UITableView, willDisplay cell: UITableViewCell, forRowAt _: IndexPath)` we establish the `dataSource` connection, maybe something like this:

```swift
let paymentMethodCell = (cell as? PledgePaymentMethodCell)
paymentMethodCell.collectionView.dataSource = self.paymentMethodsDataSource
```

(this way we keep our data sources consistently on view controllers and don't pollute table view cell with it)

Reply:
Wait... this line shouldn't be there 🤔 I think it was added by mistake, so you can ignore this. Regarding the `PaymentMethodsDataSource`, I was thinking it would be implemented once we create the `PledgeCreditCardCell`, because then we can configure the cell using the actual class using the real types to populate the cells. What do you think?